headers/types/dungeon_mode/enums.h

```diff
@@ -731,6 +731,8 @@ enum mission_type {
     MISSION_ARREST_OUTLAW = 10,
     MISSION_CHALLENGE_REQUEST = 11,
     MISSION_TREASURE_MEMO = 12,
+    MISSION_TYPE_UNK_13 = 13,
```

Comment:
Usually I prefer to use hex with an explicit `0x` in unknown enum labels to avoid ambiguity on whether a number is decimal or hex. Can we call this `MISSION_TYPE_UNK_0xD`?